src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Answers {
+    private final List<Answer> answers;
+
+    public Answers() {
+        this(new ArrayList<>());
+    }
+
+    public Answers(Answer answer) {
+        this(List.of(answer));
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+
+    public List<Answer> getAnswers() {
+        return this.answers;
+    }
+
+    public void add(Answer answer) {
+        this.answers.add(answer);
+    }
+
+    public void delete(NsUser loginUser) throws CannotDeleteException {
+        for (Answer answer : answers) {
+            answer.markAsDeleted(loginUser);
+        }
+    }
+
+    public List<DeleteHistory> createDeleteHistories() {
+        List<DeleteHistory> deleteHistories = new ArrayList<>();
+
+        for (Answer answer : answers) {
+            deleteHistories.add(new DeleteHistory(ContentType.ANSWER, answer.getId(), answer.getWriter(), LocalDateTime.now()));
```

Comment:
```suggestion
            deleteHistories.add(answer.createDeleteHistory());
```
answer의 값을 꺼내고 있는데 answer가 위와 같이 생성하도록 구현하는 것은 어떨까?